core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -675,9 +680,12 @@ public FileInfo getFileInfo(long fileId)
   public FileInfo getFileInfo(AlluxioURI path, GetStatusOptions options)
       throws FileDoesNotExistException, InvalidPathException, AccessControlException {
     Metrics.GET_FILE_INFO_OPS.inc();
-
+    Inode srcInode = null;
+    RpcUtils.RpcContext rpcContext = RpcUtils.createRpcContext().setAllowed(true);
```

Comment:
can we use `try...finally` for the `RpcContext`, similar to the journal context? The close method will do the logging.